flutter_local_notifications_windows/lib/src/details/xml/action.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:xml/xml.dart';
+import '../notification_action.dart';
+
+/// Converts a [WindowsAction] to XML
+extension ActionToXml on WindowsAction {
+  /// Serializes this notification action as Windows-compatible XML.
+  ///
+  /// See: https://learn.microsoft.com/en-us/uwp/schemas/tiles/toastschema/element-action#syntax
+  void buildXml(XmlBuilder builder) {
+    if (image != null && !image!.isAbsolute) {
+      throw ArgumentError.value(
```

Comment:
@Levi-Lesches  Just noticed this now whilst checking https://github.com/MaikuB/flutter_local_notifications/pull/2511 and noticed this. Could you shift this logic that checks this and throws an `ArgumentError` to the constructor itself? This keeps the logic co-located. Moreover, an `ArgumentError` here indicates an issue with what's passed to `buildXml()` but it's actually to do with what got passed to the constructor

Reply:
Forgot to add I commented here as there wasn't a change around this file in #2511. Happy for #2511 to include this or be in a separate PR

Reply:
Good spot! On it now